tsl/src/continuous_aggs/repair.c

```diff
@@ -130,7 +130,7 @@ cagg_rebuild_view_definition(ContinuousAgg *agg, Hypertable *mat_ht, bool force_
 							finalized,
 							NameStr(agg->data.user_view_schema),
 							NameStr(agg->data.user_view_name),
-							true);
+							false);
```

Comment:
I am wondering if we should change this also in this PR, but the old parameter value seems to be wrong.